example-data/turtle/sio-model/patientStatusDead.ttl

```diff
@@ -0,0 +1,35 @@
+@prefix : <http://purl.org/ejp-rd/cde/v020/example-rdf/> .
+@prefix obo: <http://purl.obolibrary.org/obo/> .
+@prefix sio: <http://semanticscience.org/resource/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:person1
+  a obo:NCBITaxon_9606;
+  sio:SIO_000228 :role1;
```

Comment:
It reads like you choose the use case "A person has a role, and that role participates in two processes", which is nice. But the diagram seems to assign the status and the death date into two separate processes (though I think the two tasks can be done at the same time in the same process in practice~).